### PR TITLE
removal of IPython library

### PR DESCRIPTION
**Pull request by SaraOgaz**
@larrybradley, I took out the IPython library, it is depreciated, and nbconvert works with python 2.7 and above (astropy no longer supports 2.6 I've been told).

I was messing with this code for STAK so I thought I'd put this change in here, no big deal if you don't want it.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -220,18 +220,11 @@ def notebooks_to_rst(app):
...
-            from IPython.nbconvert.preprocessors import Preprocessor, ExecutePreprocessor, execute
-            from IPython.nbconvert.exporters import RSTExporter
-            from IPython.nbformat import NotebookNode
-        except ImportError:
-            raise ImportError('Failed to find Jupyter or IPython. Cannot build '
-                              'the notebooks embedded in the docs. Proceeding '
-                              'the rest of the doc build, but additional '
-                              'warnings are likely.')
-            return
+        raise ImportError('Failed to find nbconvert. Cannot build '
```
Comment (bsipocz) [inline docs/conf.py]: Actually you can remove the whole function, as we don't have notebooks any more in this repo as don't include them either in the docs from ``photutils-datasets``

Comment (SaraOgaz) [inline docs/conf.py]: @bsipocz hah, ok, will do!

Comment (bsipocz) [inline docs/conf.py]: thanks!

**Comment by SaraOgaz** [comment]
@bsipocz are these expected Travis failures?  I can't figure out what's going off, but it looks like the same error in most of the bad runs.

**Comment by bsipocz** [comment]
@SaraOgaz - the reported failures are totally unrelated and are due to an internal travis bug, so I'm merging this now. Thanks again!

**Pull request merged**